client/my-sites/stats/stats-insights/index.jsx

```diff
@@ -69,8 +70,10 @@ export default React.createClass( {
 							beforeNavigate={ this.updateScrollPosition } />;
 		}
 
+		// TODO: should be refactored into separate components
+		/* eslint-disable wpcalypso/jsx-classname-namespace */
```

Comment:
Fair enough. :-)

My thought process was that fixing this properly involves refactoring things, not just a rename, so the warning wasn't really buying us anything. But, perhaps it buys us just a constant reminder that there is still work to do.

Since this PR was merged a month ago, if you'd like to open a new PR to get rid of the disabling, I'm cool with that.